comps/cores/proto/docarray.py

```diff
@@ -63,17 +63,16 @@ class TextImageDoc(BaseDoc):
     text: TextDoc = None
 
 
+class Base64ByteStrDoc(BaseDoc):
+    byte_str: str
+
 MultimodalDoc = Union[
+    Base64ByteStrDoc,
```

Comment:
So just remove line 70?

Reply:
Line 70 and The `Base64ByteStrDoc` can go back to where it was before. I only moved it because I was referencing it from `MultimodalDoc`, so it had to be declared before that.